components/zoom/sources/phone-event/phone-event.mjs

```diff
@@ -0,0 +1,31 @@
+import zoom from "../../zoom.app.mjs";
+import constants from "../common/constants.mjs";
+
+export default {
+  key: "zoom-phone-event",
+  name: "Zoom Phone Events",
+  description: "Listen for any Zoom Phone events tied to your Zoom user or resources you own",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    zoom,
+    eventNameOptions: {
+      label: "Zoom Events",
+      type: "string[]",
+      options: constants.PHONE_EVENT_TYPES,
+    },
+    zoomApphook: {
+      type: "$.interface.apphook",
+      appProp: "zoom",
+      async eventNames() {
+        return this.eventNameOptions;
+      },
+    },
+  },
+  async run(event) {
+    console.log(event);
+    this.$emit(event, {
+      summary: event.event,
```

Comment:
You need set `id` and `ts` here.

You can check ad fix this in others places too.

E.g: (Not tested, don't commit directly)

```suggestion
      id: event.id,
      summary: event.event,
      ts: Date.parse(event.createdAt),
```